jasmin_manage/models/service.py

```diff
@@ -57,6 +59,22 @@ def get_event_aggregates(self):
     def natural_key(self):
         return self.category.name, self.name
 
+    def get_num_active_requirements(self):
```

Comment:
In theory, you should add all of this 'row-level' functionality to the Model class rather than the manager class: https://docs.djangoproject.com/en/5.1/topics/db/managers/#adding-extra-manager-methods

(I didn't know you could add methods on the manager too which is why I went a looked it up!)

Reply:
I'm not sure I understand. Don't I need to do it here because I'm acting on all the requirements that are relevant to the service?